parm/post_avblflds.xml

```diff
@@ -6758,35 +6758,39 @@
     <param>
        <post_avblfldidx>736</post_avblfldidx>
        <shortname>SMOKE_ON_ENTIRE_ATMOS_SINGLE_LYR</shortname>
-       <pdstmpl>tmpl4_0</pdstmpl>
+       <pdstmpl>tmpl4_48</pdstmpl>
```

Comment:
@EricJames-NOAA Could you also add these tmpl4_48 updates in post_avblflds_raphrrr.xml which is used for rap/hrrr control files. After updating, you may prompt "make" under parm/ to generate new flat text control files for rap/hrrr and rtma. Please also commit these new control files.